src/Microsoft.AspNetCore.Session/baseline.netcore.json

```diff
@@ -1,601 +0,0 @@
-{
```

Comment:
I removed it because I saw it being removed in other PRs for this work. Are we not supposed to remove those files?